dev/EnvironmentManager/API/Microsoft.Windows.System.EnvironmentManager.idl

```diff
@@ -5,6 +5,10 @@
 
 namespace Microsoft.Windows.System
 {
+    [contractversion(1)]
+    apicontract WindowsSystemContract{};
```

Comment:
This is rather broad. I expected `EnvironmentManagerContract`

There was a (tactical/concrete) question about WinAppSDK contract names and API Review. Let me check on that

Reply:
https://github.com/microsoft/WindowsAppSDK/pull/2110#discussion_r821085536

@jonwis said WindowsSystemContract.

Should the naming convention be "Something that describes the type", or "Namespace"?

Reply:
I thought the naming convention was by "feature" or "feature set". Namespace is redundant, because the contract is already in a namespace, so this contract is currently named:
``` Microsoft.Windows.System.WindowsSystemContract ```

That's no good at all. I'm putting in another vote for `EnvironmentManagerContract`.

Reply:
The only item in the spec that talks about contract naming is the rule "Contract names need to end with the word `Contract`."

I like `EnvironmentManagerContract` as well.  Looks like there needs to be a talk about this.

Reply:
Contracts are units of versioning.  I'm looking for a squishy middle here - `UniversalApiContract` is much too large. `FooTypeContract` is much too small.  Other discussions suggest that a contract of 1-2 types is not interesting.

I think I'd go with `WindowsSystemContract` for now.  Ideally we get one contract per "block of functionality" ... I know that's not a satisfying answer.

Reply:
@dhoehna Is this still applicable?